pyproject.toml

```diff
@@ -34,7 +34,7 @@ include = []
 
 [tool.poetry.dependencies]
 python = "^3.7"
-gym = "^0.17.2"
+gym = ">=0.17.2"
```

Comment:
please use `^0.18` if needed. We do care about non-breaking changes from sub dependencies.